app/src/main/res/layout/activity_sign_up.xml

```diff
@@ -11,30 +11,17 @@
         android:layout_height="wrap_content"
         tools:context="org.systers.mentorship.view.activities.SignUpActivity">
 
-        <androidx.appcompat.widget.AppCompatTextView
-            android:id="@+id/tvSignUp"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:layout_marginStart="8dp"
-            android:layout_marginTop="16dp"
-            android:layout_marginEnd="8dp"
-            android:text="@string/sign_up"
-            android:textSize="24sp"
-            app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toTopOf="parent" />
-
```

Comment:
I assume you are deleting this because the `ActionBar` already contains 'Sign Up' as the title?